pkg/alpine/Dockerfile

```diff
@@ -13,9 +13,6 @@ COPY Dockerfile abuild.conf /etc/
 COPY mirrors /tmp/mirrors/
 COPY build-cache.sh /bin/
 
-RUN mkdir -vp "/mirror/edge/$(apk --print-arch)"
-# download newest go=1.20.1 from alpine/edge repository
-RUN wget -q "http://dl-cdn.alpinelinux.org/alpine/edge/community/$(apk --print-arch)/go-1.20.1-r0.apk" -O "/mirror/edge/$(apk --print-arch)/go-1.20.1-r0.apk"
```

Comment:
Do we need to leave some TBD comment somewhere? Or is the assumption that we will pick up some go-1.20* version into eve-alpine-base later?